codegen/protocol-test-codegen/build.gradle.kts

```diff
@@ -0,0 +1,39 @@
+/*
+ *     Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License").
+ *     You may not use this file except in compliance with the License.
+ *     A copy of the License is located at
+ *
+ *      http://aws.amazon.com/apache2.0
+ *
+ *     or in the "license" file accompanying this file. This file is distributed
+ *     on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *     express or implied. See the License for the specific language governing
+ *     permissions and limitations under the License.
+ */
+
+import software.amazon.smithy.gradle.tasks.SmithyBuild
+
+plugins {
+    id("software.amazon.smithy") version "0.4.3"
```

Comment:
is this version correct?

Reply:
probably not, this was copied from JS will take a look

Reply:
should probably be 1.0?